Codemine/Extensions/NSURL+AssetSize.swift

```diff
@@ -21,15 +21,15 @@ public extension URL {
      - Fit: Resizes the image to fit within the width and height boundaries without cropping or distorting the image.
      The resulting image is assured to match one of the constraining dimensions,
      while the other dimension is altered to maintain the same aspect ratio of the input image.
-     - Default: Default/normal image mode. No changes to the ratio.
+     - Standard: Default/normal image mode. No changes to the ratio.
      */
     public enum ImageUrlMode : String {
-        case Resize = "resize"
-        case Crop   = "crop"
-        case Fit    = "fit"
-        case Default = "default"
+		case resize		= "resize"
+		case crop		= "crop"
+		case fit		= "fit"
+		case standard	= "default"
```

Comment:
This should be kept as default, you can do this by writing `default` in backticks.